app/src/main/java/io/github/nicolasraoul/rosette/Bookmark.kt

```diff
@@ -0,0 +1,23 @@
+package io.github.nicolasraoul.rosette
+
+/**
+ * Data class representing a bookmarked article
+ */
+data class Bookmark(
+    val id: String,  // Wikidata ID if available, otherwise generated unique ID
+    val title: String,  // Article title
```

Comment:
no need for title and language fields, remove them.
Everything can be found from the Wikidata item.
Articles can be retitled, while Wijidata id never changes.